servlets/src/main/webapp/index-smarthome.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:height="http://java.sun.com/jsf/html" xmlns:width="http://xmlns.jcp.org/jsf/html">
+<head>
+    <meta charset="UTF-8">
+    <title>SmartHome</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+    <script src="https://use.fontawesome.com/releases/v5.0.8/js/all.js"></script>
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+<nav class="navbar-expand-md navbar-light bg-light sticky-top">
+    <!--
+Вместо navbar-light bg-light можно прописать navbar-dark bg-dark.
```

Comment:
Additional task for future - investigate how to implement it to automatically set theme based on some setting :)